### ISSUE TITLE
Coq stdlib no longer has installed .v files

### ISSUE THREAD

**Issue by JasonGross** [coq/coq#17159]
I'm not completely sure I've accurately diagnosed the problem, but it seems that the build artifacts from CI no longer(?) include `.v` files installed alongside `.vo` and `.glob` files in `_install_ci/lib/coq/theories/`, which means that coqbot CI minimization cannot inline the standard library (as desired at https://github.com/coq/coq/pull/16967#issuecomment-1348525437, cc @SkySkimmer)

**Comment by Alizter** [comment]
Yes this seems to be the case. It might be something wroth changing for 8.17.
https://github.com/coq/coq/blob/a290b310c86275123786137fd5d70963ae16e3c4/tools/dune_rule_gen/coq_module.ml#L82-L89

cc @Zimmi48 @ejgallego @coq/build-maintainers

**Comment by Alizter** [comment]
And FTR here is us installing .v files in 8.16:
https://github.com/coq/coq/blob/036eafab7934343a99d184740526675854ab977b/Makefile.install#L95

**Comment by Alizter** [comment]
And for the sake of completion. coq_makefile and dune both install the .v file also. It seems we were the odd one out here.

**Comment by ejgallego** [comment]
Yes, that's clearly a silly bug in our rule setup, I'm glad we caught it before the final release!
